package.json

```diff
@@ -12,7 +12,7 @@
   ],
   "repository": "https://github.com/liqd/adhocracy-plus.git",
   "dependencies": {
-    "@fortawesome/fontawesome-free": "6.0.0",
+    "@fortawesome/fontawesome-free": "5.15.4",
```

Comment:
This should have been done with the rebase. But if we just do a rebase soon, we can just skip this commit.